### ISSUE TITLE
[Error Handling] - Stopping a Test Run

### ISSUE THREAD

**Issue by olha23** [kubeshop/tracetest#2052]
We need allowing tests currently running to be 'canceled' or 'stopped'

### Acceptance Criteria

**AC1**
As a user that just ran a test
And I see the run is at the polling state
I should be able to stop the execution

**AC2**
As a user that just stopped a test execution
The application should provide feedback that the run was stopped
Display the section of the trace that was fetched (if any)
And the test specs shouldn't be executed

**AC3**
As a user that just stopped a test execution
The test run cannot be resumed from the point it is
To execute it again a new test run needs to be created

Mockups:
https://www.figma.com/file/LBN4SKVPq3ykegrPKbHT2Y/0.8-0.9-Release-Tracetest?node-id=1861%3A28945&t=n5GQLgd6VONU9YMK-1

**Comment by xoscar** [comment]
Clarify what needs to happen after user decides to stop a test run:
1. Should the partial trace be displayed?
2. Should we run the test specs after it?
3. Should it work as a cancel mechanism instead?


@kdhamric

**Comment by kdhamric** [comment]
Lets show what we got but not run the assertions (for now). This will allow the user to see how far things got:
- did we get a response, and what was it
- what did we get from the trace if we got anything
Seeing this info will help as you are troubleshooting response orgathering trace issues.

Should the partial trace be displayed? Yes, with a clear indication the test was stopped before the trace completed @olha23 

Should we run the test specs after it? No. We need to clearly state 'this test was manually canceled so we did not run the assertions against it' @olha23 

Should it work as a cancel mechanism instead? I don't see a lot of difference in cancel and stop. I would tend to use the word 'stop test' and show 'Test was stopped manually before completion' where necessary.

Thoughts?
